R/get_flow_data.R

```diff
@@ -4,7 +4,7 @@
 #' (if direction is "both") to and/or from the specified location.
 #'
 #' @param ef An \code{epiflows} object.
-#' @param code A character string denoting location code.
+#' @param code [OBSOLETE] A character string denoting location code.
```

Comment:
would it be better to just remove this parameter altogether? are there users who would be affected? 

not clear what the best practice is (here are some thoughts: https://ropensci.org/technotes/2017/01/05/package-evolution/) ... but having an "obosete" arg seems a little weird

Reply:
You're right, fixed